api/notd/token_metadata_processor.py

```diff
@@ -177,13 +171,14 @@ async def retrieve_token_metadata(self, registryAddress: str, tokenId: str) -> R
         # NOTE(krishan711): save the url here before using ipfs gateways etc
         metadataUrl = tokenMetadataUri
         if tokenMetadataUri.startswith('ipfs://'):
-            tokenMetadataUri = tokenMetadataUri.replace('ipfs://', 'https://ipfs.io/ipfs/')
+            tokenMetadataUri = tokenMetadataUri.replace('ipfs://', 'https://ipfs.io/')
```

Comment:
this is wrong. change back pls

Reply:
You sure?

Reply:
quite sure, what are you seeing that makes you think otherwise?

Reply:
printed out the url and clicked it and it said wait to be redirected

Reply:
like this: https://ipfs.io/ipfs/QmcUGN9woyaFpYyW99mdizM6Nqdaf5Maq2bW3TwjcgRZdS?